source/jormungandr/jormungandr/scenarios/qualifier.py

```diff
@@ -230,6 +230,22 @@ def duration_crit(j_1, j_2):
     return compare_minus(j_1.duration, j_2.duration)
 
 
+def comfort_crit(j_1, j_2):
```

Comment:
Late review (in case we come back on this): this is not a total order :-/

For example:
Journey A: 1h walk, 1 transfer
Journey B: 30min walk, 2 transfers
Journey C: 45 min walk direct
A dominates B (transfer), B dominates C (walk), C dominates A (walk).